www/docs/main/getting-started.mdx

```diff
@@ -0,0 +1,52 @@
+- [Minimal Next.js Example](https://stackblitz.com/github/trpc/trpc/tree/main/examples/next-minimal-starter?file=src%2Fpages%2Fapi%2Ftrpc%2F[trpc].ts&file=src%2Fpages%2Findex.tsx) - the smallest possible Next.js app that uses tRPC. It has a single endpoint that returns a string, and a page that calls that endpoint and displays the result.
+
+If you prefer to get started in your local environment, you can use one of our [example apps](./example-apps.mdx) as a starter project that you can experiment with locally.
+
+## Use tRPC
+
+"Using tRPC" means different things to different people. The goal of this page is to guide you to the right resources based on your goals.
+
+### Becoming productive in an existing tRPC project
+
+- Read the [concepts](./concepts.mdx) page.
+- Become familiar with [routers](../server/routers.md), [procedures](../server/procedures.md), [context](../server/context.md), and [middleware](../server/middlewares.md).
+- If you are using React, read about [useQuery](../reactjs/useQuery.md), [useMutation](../reactjs/useMutation.md) and [useContext](../reactjs/useContext.mdx).
+
+### Creating a new project
+
+Since tRPC can live inside of many different frameworks, you will first need to decide where you want to use it.
+
+On the backend, there are [adapters](../server/adapters-intro.md) for a range of frameworks as well as vanilla Node.js. On the frontend, you can use our [React](../reactjs/introduction.mdx) or [Next.js](../nextjs/introduction.mdx) integrations, a [third-party integration](../community/awesome-trpc.mdx#frontend-frameworks) for a variety of other frameworks, or the [Vanilla Client](../client/setup.mdx), which works anywhere JavaScript runs.
+
+After choosing your stack, you can either scaffold your app using a [template](./example-apps.mdx), or start from scratch using the documentation for your chosen backend and frontend integration.
+
+### Adding tRPC to an existing project
+
+Adding tRPC to an existing project is not significantly different from starting a new project, so the same resources apply. The main challenge is that it can feel difficult to know how to integrate tRPC with your existing application. Here are some tips:
+
+- You don't need to port all of your existing backend logic to tRPC. A common migration strategy is to initally only use tRPC for new endpoints, and only later migrate existing endpoints to tRPC.
+- If your project already uses Tanstack Query, make sure you don't end up with two `QueryClientProvider` components. This can happen especially in Next.js, where the `withTRPC` function already includes a QueryClientProvider of its own. If you are using `withTRPC` in your app, you should remove the `QueryClientProvider` from your `_app.tsx` file.
```

Comment:
Thought: Some of this might be worth just delegating to the specific setup guides. This level of detail on react/next I don't think belongs in the intro docs as it already exists there